FlagsmithClient/Classes/Flagsmith.swift

```diff
@@ -17,6 +17,39 @@ typealias CompletionHandler<T> = @Sendable (Result<T, any Error>) -> Void
 public final class Flagsmith: @unchecked Sendable {
     /// Shared singleton client object
     public static let shared: Flagsmith = .init()
+    
+    /// SDK version constant - should match the podspec version
+    /// This is used as a fallback when bundle version detection fails
+    private static let sdkVersionConstant = "3.8.4"
```

Comment:
I understand the resilience this adds given the platform, though this may prompt for some process that ensures code will be kept in sync with the actual version tagged.

@polatolu Do you think we need to implement some kind of CI script to accomplish this? Alternatively, would you consider falling back to "unknown" when bundle version detection fails?